azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/screens/ChatScreen.kt

```diff
@@ -126,13 +130,22 @@ internal fun ChatScreen(
             Box(modifier = Modifier.fillMaxWidth(), contentAlignment = Alignment.Center) {
                 Box(Modifier.width(ChatCompositeTheme.dimensions.messageListMaxWidth)) {
                     Column(horizontalAlignment = Alignment.CenterHorizontally) {
-                        Box(modifier = Modifier.align(alignment = Alignment.Start).padding(horizontal = 5.dp)) {
+                        Box(
+                            modifier = Modifier
+                                .align(alignment = Alignment.Start)
+                                .padding(horizontal = 5.dp)
+                        ) {
                             TypingIndicatorView(viewModel.typingParticipants.toList())
                         }
                         BottomBarView(
                             messageInputTextState = stateViewModel.messageInputTextState,
                             chatStatus = viewModel.chatStatus,
-                            postAction = viewModel.postAction
+                            postAction = {
+                                coroutineScope.launch {
+                                    listState.animateScrollToItem(0)
```

Comment:
Did you check the case when there is no item at all? Will it go produce errors?

Reply:
Not possible to have a chat with no items, as it'll have messages like "user joined"